vortex-datetime-dtype/Cargo.toml

```diff
@@ -13,6 +13,7 @@ rust-version = { workspace = true }
 
 [dependencies]
 arrow-schema = { workspace = true, optional = true }
+jiff = "0.1.8"
```

Comment:
in general we try and manage library dependencies at the workspace level, so i'd put this in the root-level Cargo.toml and then have a reference here with

```
jiff = { workspace = true }
```

Reply:
Done.